### DISCUSSION TITLE
Upvoting multiple sharedfiles?

### DISCUSSION THREAD

**Discussion by Lounas225** [3urobeat/steam-comment-service-bot#251] [Q&A] [answered]
Hello! Is it possible to upvote/favorite/funnyvote on multiple sharedfiles/reviews with one command?

**Comment by 3urobeat** [comment, marked as answer]
Hey! No, one request is always responsible for only one ID.

**Comment by oxyleti** [comment]
Can you resolve this "403" error using ai?
